include/vcpkg/base/files.h

```diff
@@ -297,8 +293,9 @@ namespace vcpkg
         void create_best_link(const Path& to, const Path& from, std::error_code& ec);
         void create_best_link(const Path& to, const Path& from, LineInfo);
 
-        virtual void copy(const Path& source, const Path& destination, CopyOptions options, std::error_code& ec) = 0;
-        void copy(const Path& source, const Path& destination, CopyOptions options, LineInfo);
+        // copies regular files and directories, recursively.
```

Comment:
We should probably note here that links are copied as if they're regular files - i.e., we copy the thing they're pointing to, not the link itself.

Reply:
That is the behavior of copy(..., copy_options::recursive), yes. Will add a note.